platform/linux/mod.rs

```diff
@@ -28,6 +28,10 @@ const MAX_FDS_IN_CMSG: u32 = 64;
 // Yes, really!
 const MAP_FAILED: *mut u8 = (!0usize) as *mut u8;
 
+// The value Linux returns for SO_SNDBUF
+// is not the size we are actually allowed to use...
+const RESERVED_SIZE: usize = 256;
```

Comment:
Can you elaborate in the comment as to how this is derived? This is kinda scary.